ops/testing.py

```diff
@@ -3053,10 +3040,8 @@ def list_files(self, path: str, *, pattern: Optional[str] = None,
             raise self._api_error(404, f"stat {path}: no such file or directory")
         files = [file_path]
         if not itself:
-            try:
+            with contextlib.suppress(NotADirectoryError):
```

Comment:
I'm pretty mixed on this. I didn't know about `contextlib.suppress` before the linter suggested it, and while this is clearly the intended use-case, I don't really understand why this is in the stdlib. Is this really better than just having the `except` with a `pass`? Am I missing an advantage of doing it this way?

Reply:
I guess the main advantage is it's 2 lines vs 4, but yeah, I'm with you -- you have to learn a new function (that has a pretty vague name -- suppress what?) and it's pretty arguable.

Reply:
Turns out this is over ten years old :laughing:. Added in https://github.com/python/cpython/issues/60010 / https://github.com/python/cpython/commit/e318a883fece5d9a45165670b5f995886f524820 (the ticket referred to in the PR is wrong, maybe that is an issue with one of the tracker migrations?).

I don't really find the arguments in the ticket convincing, so I'm going to keep not using this, and will remove it here.